.github/workflows/release-github.yaml

```diff
@@ -1,7 +1,7 @@
 name: Build release
```

Comment:
Can this name get more specific as well?

Reply:
Done.